site/_includes/partials/chrome-100-footer-section.njk

```diff
@@ -0,0 +1,20 @@
+{% block content %}
+<section class="footer footer-top-section display-flex direction-column justify-content-center">
+  <img src="/images/project/no-internet.svg" alt="No internet logo" class="no-internet-logo">
+  <div class="text-align-center footer-description gap-top-500">
+    <div class="footer-subheading">#100CoolWebMoments</div>
+    <p class="footer-info">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
```

Comment:
I'm assuming you're aware that this need to be swapped out. Just pointing it out in case.

Reply:
Swapped